sim-lib/src/random_activity.rs

```diff
@@ -348,26 +382,50 @@ mod tests {
             // distribution must fail building given the inputs. The former will be thoroughly tested in its own unit test, but we'll test some basic cases
             // here. Mainly, if the `capacity < expected_payment_amnt / 2`, the generator will fail building
             let expected_payment = get_random_int(1, 100);
-            assert!(
-                RandomPaymentActivity::new(2 * expected_payment, expected_payment, 1.0).is_ok()
-            );
+            assert!(RandomPaymentActivity::new(
+                2 * expected_payment,
+                expected_payment,
+                1.0,
+                Some([42; 32])
```

Comment:
Any specific reasoning for using `None` above and `Some` here for seeds? non-blocking

Reply:
The seed parameter is optional so I thought one test could run with it provided, and another without it. If consistency matters to you I can write a second string of tests for both cases.

Reply:
> The seed parameter is optional so I thought one test could run with it provided, and another without it. If consistency matters to you I can write a second string of tests for both cases.

Was just wondering if there was any reason for the choice for those specific tests using the specific params, but makes sense!

> What are you referring to with "non-blocking"?

"non-blocking" for merge - so I have a question here but it's not something that _must_ be addressed/answered for the PR to move forward, just asking for my own understanding 👍